### PR TITLE
L1 layout

### LINKED ISSUE #272: Featured Articles
Article cards in horizontal layout

Image

### PR DESCRIPTION
**Pull request by saurabh-khare**
Preliminary layout for L1 page which includes:

1. L1 page template
2. Featured cards in stack layout
3. Single column text only list

Fix #272, #219 

Test URLs:
- Before: https://main--hlx-test--urfuwo.hlx.page/draft/rootpea/219-l1-sample-page
- After: https://272-featured-articles--hlx-test--urfuwo.hlx.page/draft/rootpea/219-l1-sample-page

### CODE REVIEW

**Review by rootpea — APPROVED** [review]
looks good, please address the linter comments before merging

**Review by pahupe — COMMENTED** [review]
General question: how much overlapping of l1 and l2 is there? If the overlapping is high, maybe we can refactor it

**Comment by saurabh-khare** [comment]
> General question: how much overlapping of l1 and l2 styles is there? If the overlapping is high, maybe we can refactor the CSS

I would say for now still a high degree but L1 is just in infancy so I expect L1 template to diverge from L2 and we anyways don't want to send extra bytes for L2 styles to L1 page. But yes, I agree we should revisit later point in time if we can merge common styles.

**Comment by mhaack** [comment]
> we should revisit later point in time

Let's do the opposite, unify now. Lets not add stuff we "maybe need later". YANGI & KISS are our friends.

**Comment by saurabh-khare** [comment]
> > we should revisit later point in time
> 
> Let's do the opposite, unify now. Lets not add stuff we "maybe need later". YANGI & KISS are our friends.

Looking at the L1 design we might be adding more L1 specific styles in same template which will maintain both styles so we'll be sending extra bytes for styles not required on the page but sure we can separate the template later. But that will also require changing the page metadata to change the template if required. 
For now I'll merge the templates.

**Comment by mhaack** [comment]
> For now I'll merge the templates

+1 if we later see a case we can carve out thinks. Also changing metadata for one or 2 pages is not an issue

File: aemedge/templates/hub-l1/hub-l1.css
[the remark this answers is not in the document]
Comment (benpeter) [reply aemedge/templates/hub-l1/hub-l1.css:1]: is this resolved @mhaack ?

**Review by mhaack — APPROVED** [review]

**Pull request merged**